packages/mdc-tab-scroller/_mixins.scss

```diff
@@ -35,6 +57,8 @@
     @include mdc-feature-targets($feat-structure) {
       overflow-y: hidden;
     }
+
+    @include mdc-tab-scroller-transition($mdc-tab-scroller-transition-duration, $query)
```

Comment:
Move this line before above include statement.

Reply:
Done.